single_kernel_mongo/managers/mongos_operator.py

```diff
@@ -408,6 +393,24 @@ def is_mongos_running(self) -> bool:
 
         uri = f"mongodb://{host}"
 
-        return self.mongo_manager.mongod_ready(uri=uri, direct=False)
+        return self.mongo_manager.mongod_ready(uri=uri)
+
+    def get_status(self) -> StatusBase | None:
```

Comment:
I think this name can be deceptive, maybe `get_sanity_check_statuses` with a docstring that says "Retrieve statuses that directly relate to states of mongos which would prevent other more advanced mongos statuses from being checked."